applications/convection_diffusion_application/python_scripts/proposal_convection_diffusion_solver.py

```diff
@@ -0,0 +1,271 @@
+from __future__ import print_function, absolute_import, division
```

Comment:
i already asked you what is this "proposal_" about...

Reply:
Is the one added by @jcotela, I restored the old one, because without it the old tests don't pass

Reply:
Why do we even need it? You already added three completely new solvers. (BTW, some tests in the fluid dynamics are broken, since I changed them in #2214 to use my draft, and these changes have somehow ended up in the release-6.0 branch)